views/templates/admin/smaily_configure.tpl

```diff
@@ -181,6 +181,38 @@
               </p>
             </div>
           </div>
+          <div class="form-group">
+            <label class="control-label col-lg-2">
+              {l s="Trigger Opt-in If Customer Joins With Newsletter Subscription" mod='smailyforprestashop'}
```

Comment:
Improve phrasing. Use something like:

> Trigger opt-in on customer sign-up

And add help block under the slide button (like here - https://github.com/sendsmaily/smaily-prestashop-module/blob/master/views/templates/admin/smaily_configure.tpl#L157-L159):

> Note! Opt-in will only be triggered when customer signs-up for newsletter. Changes to newsletter subscription in admin won't trigger an opt-in.

Reply:
Renamed and added help block